test/TestExtensions/TestablePlugin/TestablePlugin.csproj

```diff
@@ -21,6 +21,13 @@
     <ProjectReference Include="$(NuGetCoreSrcDirectory)NuGet.Versioning\NuGet.Versioning.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IsCore)' == 'true'">
+    <PackageReference Include="System.Collections" Version="4.3.0" />
```

Comment:
I'll surprised this did not fail the build. 

The versions should remain managed in packages.targets. 

There's a section with packages used in test.

Reply:
I'm also surprised it didn't fail the build, but it didn't. This suggests that removing the versions might leave these `PackageReference`s without a version, in which cause the build might fail. So, my thought is we merge as-is, and we should fix our msbuild files as an engineering improvement, and not burden the contributor for this.

Reply:
I buy that. 
Can we create a follow-up issue first?

Reply:
Too late, I used the `SystemPackagesVersion` property.

If I recall correctly, these references stayed because they resolve package version conflicts.

Reply:
https://github.com/NuGet/Home/issues/9790

Reply:
I tried it again to build without these references anf got this (adjusted for brevity):

```
error NU1605: Detected package downgrade: System.Collections from 4.3.0 to 4.0.11. Reference the package directly fr
om the project to select a different version.
error NU1605:  Plugin.Testable -> NuGet.Protocol -> NuGet.Packaging -> Newtonsoft.Json 9.0.1 -> System.Xml.ReaderWri
ter 4.0.11 -> System.IO.FileSystem 4.0.1 -> runtime.win.System.IO.FileSystem 4.3.0 -> System.Collections (>= 4.3.0)
error NU1605:  Plugin.Testable -> NuGet.Protocol -> NuGet.Packaging -> Newtonsoft.Json 9.0.1 -> System.Collections (
>= 4.0.11)
```

Same with the other packages. The real fix was to upgrade `Newtonsoft.Json` but unfortunately it can't be done at the moment.

Reply:
@zivkan, it didn't fail the build because the `SystemPackagesVersion` property is equal to `4.3.0`.

Reply:
As per the issue I linked, our MSBuild targets are supposed to validate that no `PackageReference` has a `Version` attribute set before it then applies all the versions. This isn't working, but is out of scope for your PR.  Assuming the current build in progress is green, I'll merge this PR and the NuGet team will be responsible for fixing up our MSBuild files (unless someone in the community wants to do it and send a PR 😊  ).